modules/runner/src/integrationTest/java/org/apache/ignite/internal/schemasync/ItSchemaSyncAndReplicationTest.java

```diff
@@ -145,12 +149,8 @@ private CompletableFuture<?> rejectionDueToMetadataLagTriggered() {
         return rejectionTriggered;
     }
 
-    private void putToTableAt(int nodeIndex) {
-        cluster.node(nodeIndex)
-                .tables()
-                .table(TABLE_NAME)
-                .keyValueView()
-                .put(null, Tuple.create().set("key", 1), Tuple.create().set("val", "one"));
+    private void putToTableAt(Table table) {
```

Comment:
`at` made sense when the parameter was an index, now I suggest to drop the suffix so that the method name becomes 'putToTable()` (or even `putTo()`)